numba/tests/test_unicode.py

```diff
@@ -74,6 +76,25 @@ def endswith_usecase(x, y):
     return x.endswith(y)
 
 
+def split_usecase(x, y):
+    return x.split(y)
+
+
+def split_with_maxsplit_usecase(x, y, maxsplit):
+    return x.split(y, maxsplit)
```

Comment:
A variation on this with `maxsplit` explicitly declared as a kwarg would be good to test.

Reply:
added this test